### DISCUSSION TITLE
Webtransport

### DISCUSSION THREAD

**Discussion by unicomp21** [microsoft/msquic#2302] [Q&A] [answered]
https://blog.chromium.org/2021/11/chrome-97-webtransport-new-array-static.html

Given webtransport just recently went GA, is there an example anywhere showing a browser client hitting msquic?

**Comment by nibanks** [comment, marked as answer]
I would LOVE to have a OSS WebTransport solution on top of MsQuic. But WebTransport first requires a minimal HTTP/3 layer. My team hasn't had the time or resources to implement this. There is .NET's C# HTTP/3 on top of MsQuic, but it's obviously not C/C++. Until we, or someone else, is able to put this together, you won't be able to use MsQuic for WebTransport.

**Comment by unicomp21** [comment]
Is there another .Net options for handling webtransport server side connections?

**Reply by nibanks** [reply]
No, not currently (as far as I'm aware).
